src/number_theory/quadratic_reciprocity.lean

```diff
@@ -356,13 +360,13 @@ namespace zmod
 * `-1` otherwise.
 
 -/
-def legendre_sym (a : ℤ) (p : ℕ) : ℤ :=
+def legendre_sym (p : ℕ) (a : ℤ) : ℤ :=
```

Comment:
Please also include a warning about the order of the arguments in the docstring. (I saw you already added that warning above in the module docs.)

Reply:
Thanks; I had overlooked that. I have changed the docstring accordingly.